app/frontend/Calendar/DisplayEvent/Invitation.svelte

```diff
@@ -76,7 +76,13 @@
         </ButtonMenu>
       {/if}
     {:else if $message.invitationMessage && incomingInvitation}
-      <Button label={$t`Update calendar`} disabled={updateDisabled} onClick={onUpdate} classes="font-normal" />
+      {#await onUpdate()}
```

Comment:
`onUpdate` will fail if we didn't find a calendar. Also, what happens if we find multiple calendars?

Reply:
fixed, I think